game.js

```diff
@@ -0,0 +1,121 @@
+var sizeOfOneSquare = 40, sizeOfFoundation = 3, canvas =document.getElementById("main"), context = canvas.getContext("2d");
+function drawmap() {
+    var i;
+    for (i=0; i<sizeOfFoundation; i+=1){
+        for (var j=0; j<sizeOfFoundation; j+=1){
+            context.strokeRect(i*sizeOfOneSquare, j*sizeOfOneSquare,sizeOfOneSquare, sizeOfOneSquare);
+        }
+    }
+}
+drawmap();
+var WidthOfWindow =  window.innerWidth*0.45, HeightOfWindow =window.innerHeight*0.2, flagForChangeStep = true, flagWinner = false,arrayForO = [], arrayForX = [], countSteps = 0;
```

Comment:
use the same numbers of tabs and spaces. IN the whole file.
i.e 
var WidthOfWindow =  window.innerWidth*0.45, HeightOfWindow = window.innerHeight*0.2, flagForChangeStep = true, flagWinner = false,arrayForO = [], arrayForX = [], countSteps = 0;